### PR TITLE
feat(electron): enable vscode debugger for electron

### PR DESCRIPTION
**Pull request by pengx17**

### CODE REVIEW

**Comment by pengx17** [comment]
* **#9333** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9333?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9333?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#9328** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9328?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `canary`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/toeverything?ref=merge-queue-instructions-in-stack-comment&prId=9333).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: tools/utils/src/path.ts [hunk 1]
```diff
@@ -42,7 +42,8 @@ export class Path {
   }
 
   toFileUrl() {
-    return pathToFileURL(this.path);
+    const url = pathToFileURL(this.path).toString();
+    return url.startsWith('file://') ? url : `file://${url}`;
```
Comment (forehalo) [inline tools/utils/src/path.ts]: wtf? toFileUrl will not always prefix the 'file://'?

Comment (pengx17) [reply tools/utils/src/path.ts]: Two days ago I encountered an error on running `affine` commands that are lacking of `file://` prefix. I cannot reproduce the issue any longer 🤔

Comment (forehalo) [reply tools/utils/src/path.ts]: I saw it happened on windows once and this function is meant to fix it. the problem must be somewhere else

File: tools/cli/src/run.ts [hunk 2]
```diff
@@ -151,12 +151,18 @@ export class RunCommand extends PackageCommand {
       new RegExp(ignore).test(scriptName)
     );
 
+    const loaderOption =
```
Comment (forehalo) [inline tools/cli/src/run.ts]: done in #9328

**Review by doodlewind — APPROVED** [review]

**Review by darkskygit — APPROVED** [review]

**Pull request merged**